### PR TITLE
feat(connector): added VenlyConnector

### PR DESCRIPTION
**Pull request by davidzwfu**
## Description

Hello, David from the Venly team here. This PR adds the `VenlyConnector` package for our [Venly wallet](https://www.venly.io/product/wallet-solutions).

The Connector presents a novel use-case as it utilizes our own SDK (similar to CoinbaseWalletConnector). 
Link to our connector request can be found [here](https://github.com/wagmi-dev/references/discussions/202).

## Additional Information

- [x] I read the [contributing docs](/wagmi-dev/references/blob/main/.github/CONTRIBUTING.md) (if this is your first contribution)

### CODE REVIEW

File: packages/connectors/package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@web3modal/standalone": "^2.3.0",
     "@safe-global/safe-apps-provider": "^0.15.2",
     "@safe-global/safe-apps-sdk": "^7.9.0",
+    "@venly/web3-provider": "^3.0.0-develop.8",
```
Comment (jxom) [inline packages/connectors/package.json:31]: Is this a stable version?

Comment (davidzwfu) [reply packages/connectors/package.json]: Updated to latest stable release

**Comment by davidzwfu** [comment]
Hello wagmi team, amazing work so far on viem and the v1 release.

I've updated the VenlyConnector to match the v1 changes, as well as updated to our latest release build. Please have a look when you get the chance.

**Comment by davidzwfu** [comment]
@jxom @tmm friendly ping for a status update on this PR. This integration is particularly important for us because of our partnership with Shopify (their connect-wallet component is built on wagmi). Would be very much appreciated

**Comment by tmm** [comment]
@davidzwfu thanks for the ping! I recommend publishing your own npm package for this connector until we have a moment to review. There are a few other areas we are working on at the moment.

**Comment by davidzwfu** [comment]
> @davidzwfu thanks for the ping! I recommend publishing your own npm package for this connector until we have a moment to review. There are a few other areas we are working on at the moment.

@tmm Thanks for getting back to me. Do you have an approximate eta perhaps? Would help me to form somewhat of a timeline.

**Review by aiiiden — COMMENTED** [review]
image
따로 형태감은 잡아놓으면 좋을 것 같아요

**Comment by tmm** [comment]
Appreciate the effort that went into this! We've been busy with other work so haven't had time to support new connectors. We are reworking how connectors are setup (directly in the wagmi repo instead of references, new API, etc.) so once that is done this can be created again. Stayed tuned to the [wagmi discussion board](https://github.com/wagmi-dev/wagmi/discussions) for more info.

**Pull request closed without merging**